package.json

```diff
@@ -33,7 +33,8 @@
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
   "package-json-lint": {
-    "extends": "./packages/package-json-lint-config-terra/package-json-lint.config.js"
+    "extends": "./packages/package-json-lint-config-terra/package-json-lint.config.js",
+    "projectType": "module"
```

Comment:
This should be a devModule

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/704/commits/cc6d1e0cc26383d90b787febbf2eeec56d41464a.